docusaurus.config.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
     footer: {
       logo: {
         alt: 'Tremor Logo',
-        src: 'static/favicon.ico',
+        src: '/img/favicon.ico',
```

Comment:
this should probably just be `/favicon.ico`

Reply:
Oh, I didn't know that. Thanks.